docs/guides/audio-config.md

```diff
@@ -20,33 +20,22 @@ If not present, you'll have to update your bootup kernel params:
 
 # Audio Configuration Files
 
-In most scenarios, you should use [these files](https://gist.github.com/MCMrARM/c357291e4e5c18894bea10665dcebffb), following the instructions in that gist's `README.md`.
+Firstly, check whether you are using an updated apple-bce version by running `sed -n "s/.*\(AppleT2.*\) -.*/\1/p" /proc/asound/cards`. If the output is `AppleT2xN` (where `N` is a number), then this guide can be followed. If its just `AppleT2`, then either update the driver first or follow the [older version](https://github.com/t2linux/wiki/blob/00e882d0d5afb3102aedd02872426dc5fba789a5/docs/guides/audio-config.md) of this guide.
 
-Special scenarios are:
+Simply run the following to set up audio :-
 
-- [2019 16" MacBook Pro audio files](https://gist.github.com/kevineinarsson/8e5e92664f97508277fefef1b8015fba) - that laptop has 6 speakers and needs slightly different config
-- [2020 13" MacBook Air audio files](https://gist.github.com/bigbadmonster17/8b670ae29e0b7be2b73887f3f37a057b)
+```bash
+git clone https://github.com/kekrby/t2-better-audio.git
+cd t2-better-audio
+bash ./install.sh
+sudo rm -r $HOME/t2-better-audio
+```
 
 # Using Pipewire instead of Pulseaudio
 
-You'll need to modify the `/lib/udev/rules.d/91-pulseaudio-custom.rules` file from the links above.
-
-```diff
-SUBSYSTEM!="sound", GOTO="pulseaudio_end"
-ACTION!="change", GOTO="pulseaudio_end"
-KERNEL!="card*", GOTO="pulseaudio_end"
-
--SUBSYSTEMS=="pci", ATTRS{vendor}=="0x106b", ATTRS{device}=="0x1803", ENV{PULSE_PROFILE_SET}="apple-t2.conf"
-+SUBSYSTEMS=="pci", ATTRS{vendor}=="0x106b", ATTRS{device}=="0x1803", ENV{PULSE_PROFILE_SET}="apple-t2.conf", ENV{ACP_PROFILE_SET}="apple-t2.conf"
-
-LABEL="pulseaudio_end"
-```
-
-Note: The updated locations for the files in the links above for Pipewire distributions will be:
+Due a bug in PulseAudio, the current configuration does not work with it. A fix has been [already merged](https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/596) but a new version containing the changes hasn't been released yet, so you should use PipeWire instead of PulseAudio unless your distro builds PulseAudio from the main branch or applies the patch before building.
 
-- /usr/share/alsa/cards/AppleT2.conf
-- **/usr/share/alsa-card-profile/mixer/profile-sets/apple-t2.conf**
-- /usr/lib/udev/rules.d/91-pulseaudio-custom.rules
+Different distros have different ways to set up Pipewire. An example to get Pipewire working on Ubuntu, which uses Pulseaudio by default is given [here](https://linuxconfig.org/how-to-install-pipewire-on-ubuntu-linux).
```

Comment:
I think it should be mentioned earlier that currently only `pipewire` works (without patching) and this should be put after that.